leaf-ui/css/style.css

```diff
@@ -45,6 +45,36 @@ body.waiting *
   font-style: italic;
 }
 
+/** experimental dim screen */ 
```

Comment:
Is it still experimental? If so, why keep it?

Reply:
> Is it still experimental? If so, why keep it?

Deleted as we don't need it anymore.